data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/schema/GetBlockResponse.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.beaconrestapi.schema;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import tech.pegasys.teku.api.schema.SignedBeaconBlock;
+
+public class GetBlockResponse {
+
+  @JsonProperty("beacon_block")
+  public final SignedBeaconBlock signedBeaconBlock;
+
+  public final String root;
+
+  public GetBlockResponse(final SignedBeaconBlock signedBeaconBlock) {
+    this.signedBeaconBlock = signedBeaconBlock;
+    this.root =
```

Comment:
Any chance we can avoid recreating the original block just to compute the root?

Reply:
yep i just need to work the response through the provider layer which i wasn't super keen on... i'll take a closer look.